.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{html,css,js}]
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+[*.go]
+indent_style = tab
+indent_size = 8
```

Comment:
That is what i meant. It doesn't affect the code if you use 4 in your editor and I use 8 in mine, thus it shouldn't be forced in an `.editorconfig`.